packages/server/bootstrap.js

```diff
@@ -5,11 +5,11 @@
 
 // Initializing module aliases for absolute import paths
 require('module-alias')({ base: __dirname })
+const appRoot = __dirname
 
 // Initializing env vars
 const dotenv = require('dotenv')
 const { isTestEnv } = require('./modules/core/helpers/envHelper')
-const appRoot = require('app-root-path')
 
 // If running in test env, load .env.test first
 // (appRoot necessary, cause env files aren't loaded through require() calls)
```

Comment:
Maybe we can get rid of appRoot here and just use `require.resolve` to resolve the path to the file, like you've done elsewhere 🤔

Reply:
yeah i've pused an, we'll need the appRoot in `server/modules/index.js` line 34 anyhow